modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -56,24 +56,37 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
      */
     function _setupVariables()
     {
+        $config = NDB_Config::singleton();
+        $DB = Database::singleton();
+        $scan_types_to_table = $config->getSetting('tblScanTypes');
+        $scan_types_to_table = explode(',', $scan_types_to_table);
```

Comment:
this is now changed to get values like it is done for instruments.
I set the default in the patch to populate for t1 and t2 (ID=44 and 45) so existing projects will see no change. But I will need to handle the case when the user specifically erases t1 and t2 from Configuration and leaves it blank (in which case the table in Imaging Browser will be without these columns).